src/style.css

```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.moyo-card {
+  box-sizing: border-box;
```

Comment:
move` box-sizing: border-box;` to * selector

Reply:
i corrected, but i am not sure that i understood you correctly